lib/dotcom_web/router.ex

```diff
@@ -265,7 +265,7 @@ defmodule DotcomWeb.Router do
 
   scope "/preview", DotcomWeb do
     import Phoenix.LiveView.Router
-    pipe_through([:browser, :browser_live, :basic_auth_readonly])
+    pipe_through([:browser, :browser_live])
```

Comment:
📓 Just did this for convenience since basic auth won't be there in the near future. Will remove this from the PR before merge.

Reply:
Why not remove it now?

Reply:
I'm going to in a single PR with all of these dangling trip planner tasks.